2020/scoring/README.md

```diff
@@ -0,0 +1,31 @@
+### Compute the scores for the website by combining different jobs
+
+Once results are downloaded form starexec, the scores are computed in
```

Comment:
Add a paragraph with instructions for downloading:

First you need to download the "job information" from StarExec.  Make sure the option "include ids" is checked.  If the Job is public, you can download using curl:

```sh
curl -o Job${JOBID}_info.zip "https://www.starexec.org/starexec/secure/download?type=job&id=${JOBID}&returnids=true"
```

Reply:
Ok/